File: api/film/film.java
package com.oguzhan.api.film;

import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.Version;

import java.sql.Timestamp;
import java.time.LocalDateTime;

public record film (
    @Id
    Integer id,
    String isim,
    Integer uzunluk,
    String konu,
    Integer degerlendirme_sayisi,
    Float ortalama_puan,
    String poster_uri
){}

File: odevproje/model/film.kt
package com.example.odevproje.model

data class film (
    val id: Int,
    val isim: String,
    val uzunluk: Int,
    val konu: String,
    val degerlendirme_sayisi: Int,
    val ortalama_puan: Double,
    val poster_uri: String
)